waiter/config-full.edn

```diff
@@ -823,9 +823,15 @@
                  ;; The interval (milliseconds) on which Waiter makes work-stealing offers:
                  :offer-help-interval-ms 100
 
+                 ;; The duration (milliseconds) after which Waiter times out work-stealing offers:
+                 :offer-idle-timeout-ms 1800000
```

Comment:
This seems like an too large idle timeout.  We keep the connection open the entire time the offer is being made?

Reply:
Yes, the connection stays open for the duration of the offer.
Changed it back to 15 minutes.